modules/widgets/facebook-likebox.php

```diff
@@ -168,15 +168,19 @@ function form( $instance ) {
 		<p>
 			<label for="<?php echo esc_attr( $this->get_field_id( 'width' ) ); ?>">
 				<?php _e( 'Width in pixels', 'jetpack' ); ?>
-				<input type="number" class="smalltext" min="<?php echo esc_attr( $this->min_width ); ?>" max="<?php echo esc_attr( $this->max_width ); ?>" maxlength="3" name="<?php echo esc_attr( $this->get_field_name( 'width' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'width' ) ); ?>" value="<?php echo esc_attr( $like_args['width'] ); ?>" style="text-align: center;" />
+				<input type="number" class="smalltext" min="0" max="<?php echo esc_attr( $this->max_width ); ?>" maxlength="3" name="<?php echo esc_attr( $this->get_field_name( 'width' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'width' ) ); ?>" value="<?php echo esc_attr( $like_args['width'] ); ?>" style="text-align: center;" />
```

Comment:
I don't know what to think about this. I think this could be confusing for some, especially folks using a screen reader. They are now able to enter low values in that field, and may not necessary see that this value was not accepted when the widget got saved.

Reply:
Agreed. I was initially going for the checkbox idea you suggested above. I tried something **similar** to what we have in the [contact widget](https://github.com/Automattic/jetpack/blob/af7e183ff050150a8f08161cc564771c744f6598/modules/widgets/contact-info.php) - 

This `checked( $instance['showmap'], 1 );` 
followed by `style="<?php echo $instance['showmap'] ? '' : 'display: none;'; ?>`

But it didn't update the form immediately as I checked the button. Instead the form got updated and disabled the width input "after" I saved the widget.

I'm going to try this again but if you have any quick suggestion for me, please let me know! I appreciate all your help :)